package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "cypress",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run cytest"
```

Comment:
Please update the script.

you can use anything you want, this will be the script you run on the CI and you can also run local as a shortcut.

example is:

```
"test": "npx cypress run --headless",
"debug": "npx cypress open" 
```

must use headless for CI

so then you can use 

```
npm run test
npm run debug
```